getting-started/frames.ipynb

```diff
@@ -990,102 +991,88 @@
-    "- **An SQL query** \\[Tech Preview\\] &mdash; set the `query` parameter to an SQL query string of the following format:\n",
-    "  ```\n",
-    "  select <metrics | aggregators> from <table path> [where <filter expression>] [group by <labels>]\n",
-    "  ```\n",
-    "  > **Note:**\n",
-    "  > - The `where` filter expression is similar to that passed to the `filter` parameter for a non-SQL query, except it's in SQL format, so the expression isn't embedded within quotation marks and comparisons are done by using the '`=`' operator instead of the '`==`' operator.\n",
-    "  > - The `select` clause can optionally include a comma-separated list of either over-time aggregators (such as `avg` or `sum`) or cross-series aggregators (such as `avg_all` or `sum_all`), but you cannot mix these aggregation types.\n",
-    "  >   The aggregation functions receive a metric-name parameter (for example, `avg(cpu)`, `avg_all(cpu)`, or `avg(*)` for all metrics).\n",
-    "  >   Cross-series aggregations functions can also optionally receive an interpolation function &mdash; `next` (default) | `prev` | `linear` | `none` &mdash; in which case the metric name is passed as a parameter of the interpolation function (and not as a direct parameter of the aggregation function); the interpolation function can also optionally receive an interpolation-tolerance string of the format `\"[0-9]+[mhd]\"` (for example, `avg_all(prev(cpu,'1h'))`).\n",
     "\n",
-    "For both types of queries, you can also optionally set additional parameters.\n",
-    "`start` and `end` define the query's time range &mdash; the metric-sample timestamps to which to apply the query (the default end time is `\"now\"` and the default start time is 1 hour before the end time); `step` defines the interval for aggregation or raw-data downsampling (default = the query's time range); and`aggregationWindow` defines the aggregation time frame for over-time aggregation (default = the aggregation step).<br>\n",
-    "You can set the optional `multi_index` parameter to `True` to return labels as index columns, as demonstrated in the following example.\n",
-    "By default, only the metric sample-time primary-key attribute is returned as an index column.<br>\n",
-    "See the [Frames documentation](https://github.com/v3io/frames/blob/master/README.md) for more information about the supported parameters of the `read` method for the `tsdb` backend."
+    "# Genearte a DataFrame with TSDB metric samples and a \"time\" index column\n",
+    "def gen_df_w_tsdb_data(num_items=24, freq=\"1H\", end=None, start=None,\n",
```

Comment:
if you created the table with `1/m` rate better create the data the same way (with 1m freq) for consistency.

Reply:
@talIguaz as we agreed f2f, I changed the `rate` in the `create` example to `1h` to match the write data set.